### PR TITLE
[1LP][RFR] Fix automate nav by using icons

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: -v cfme/tests/automate/test_instance.py cfme/tests/automate/test_method.py -k crud --long-running}}

### CODE REVIEW

**Comment by mfalesni** [comment]
Almost there, just needs the tree path matching in is_displayed to count with the icon ...

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/automate/explorer/instance.py [hunk 1]
```diff
@@ -196,10 +196,15 @@ def domain(self):
 
     @property
     def tree_path(self):
+        if self.appliance.version < '5.8':
```
Comment (otsuman) [inline cfme/automate/explorer/instance.py]: No need for this, as I understood we don't support 5.7 now

File: cfme/automate/explorer/method.py [hunk 2]
```diff
@@ -157,10 +157,17 @@ def domain(self):
 
     @property
     def tree_path(self):
+        if self.appliance.version < '5.8':
```
Comment (otsuman) [inline cfme/automate/explorer/method.py]: same here

File: cfme/automate/explorer/instance.py (hunk 1 above)
Comment (mfalesni) [reply cfme/automate/explorer/instance.py]: Yeah, I will remove it

**Pull request merged**